enarxbot-triage

```diff
@@ -82,16 +82,21 @@ if bot.PROJECTS["Planning"] not in projects:
     print(f"Adding {owner}/{repo}#{result['node']['number']} to Planning board")
 
     # Add the content to the project.
-    bot.graphql(
-        """
-        mutation($input:AddProjectCardInput!) {
-            addProjectCard(input:$input) {
-                clientMutationId
+    try:
+        bot.graphql(
+            """
+            mutation($input:AddProjectCardInput!) {
+                addProjectCard(input:$input) {
+                    clientMutationId
+                }
             }
-        }
-        """,
-        input={
-            "projectColumnId": bot.COLUMNS["Planning"]["Triage"],
-            "contentId": id
-        }
-    )
+            """,
+            input={
+                "projectColumnId": bot.COLUMNS["Planning"]["Triage"],
+                "contentId": id
+            }
+        )
+    except bot.GraphQLError as e:
+        if e.errors[0]["message"] != "Project already has the associated issue":
```

Comment:
Are these messages defined as constants somewhere?

Reply:
We don't throw that specific string -- it's returned by Github, and we just check for it to determine if it's something that should be a soft failure.

We could put the string in a common place somewhere (maybe `bot.py`), I suppose.